libs/expo/betterangels/src/lib/static/enumDisplayMapping.ts

```diff
@@ -173,26 +173,37 @@ export const clientHouseholdMemberEnumDisplay: Partial<{
 };
 
 export const enumDisplayServices: { [key in ServiceEnum]: string } = {
+  [ServiceEnum.Bag]: 'Bag(s)',
+  [ServiceEnum.Batteries]: 'Batteries',
   [ServiceEnum.Blanket]: 'Blanket',
   [ServiceEnum.Bicycle]: 'Bicycle',
+  [ServiceEnum.BicycleRepair]: 'Bicycle Repair',
   [ServiceEnum.BirthCertificate]: 'Birth Certificate',
   [ServiceEnum.Book]: 'Book',
   [ServiceEnum.CaliforniaLifelinePhone]: 'California Lifeline Phone',
   [ServiceEnum.Clothes]: 'Clothes',
+  [ServiceEnum.ClothesSocks]: 'Socks - Clothes',
```

Comment:
I do not think socks - clothes should be here based off previous discussions to collapse categories.